test/render_html.py

```diff
@@ -45,24 +45,24 @@ def render_html(themes: list, languages=languages, outdir=outdir):
     if not outdir.exists():
         os.mkdir(outdir)
 
-    for language in languages:
-        ext = languages[language]
-        name = HERE / "scripts" / f"test.{ext}"
+    for theme in themes:
 
-        with open(name, "r") as f:
-            lines = f.read()
+        theme_outdir = outdir / theme
+        if not theme_outdir.exists():
+            os.mkdir(theme_outdir)
 
-        lexer = get_lexer_by_name(language, stripall=True)
+        style = get_style_by_name(theme)
+        formatter = HtmlFormatter(style=style, full=True, hl_lines=[2, 3, 4])
```

Comment:
Adding a reference here for ease of contribution and such
```suggestion
        #ref: https://pygments.org/docs/formatters/#HtmlFormatter
        formatter = HtmlFormatter(style=style, full=True, hl_lines=[2, 3, 4])
```